src/status_im/components/button/styles.cljs

```diff
@@ -36,16 +36,17 @@
          (border position)))
 
 (defnstyle button-text [disabled?]
-  {:font-size          15
-   :font-weight        "normal"
+  {:font-weight        "normal"
    :color              styles/color-white
    :padding-horizontal 16
    :android            (merge
-                         {:padding-vertical   10
-                          :letter-spacing     0.2}
+                         {:font-size          14
+                          :padding-vertical   10
+                          :letter-spacing     0.5}
```

Comment:
last time I checked letter-spacing doesn't work with react-native on Android

Reply:
Right, see #2045. But it might at some point so better add it now.